lib/bootloader_pvm.pm

```diff
@@ -130,7 +130,7 @@ sub prepare_pvm_installation {
     my $disks = script_output('lsblk -n -l -o NAME -d -e 7,11');
     for my $d (split('\n', $disks)) {
         script_run "wipefs -a /dev/$d";
-        create_encrypted_part(disk => $d) if get_var('ENCRYPT_ACTIVATE_EXISTING');
+        create_encrypted_part(disk => $d) if get_var('ENCRYPT_ACTIVATE_EXISTING') || get_var('ENCRYPT_CANCEL_EXISTING');
```

Comment:
we could document this variable, although I see that was missing before you use it.

Reply:
Done.